JavaBytecodeCompiler/src/main/java/org/openzen/zenscript/javabytecode/compiler/JavaMethodBytecodeCompiler.java

```diff
@@ -214,10 +214,19 @@ private void handleArguments(int typeArguments, JavaNativeMethod method, CallArg
 		if (method.compile) {
 			handleTypeArguments(method, arguments);
 		}
+
+		// This happens e.g. for Strings where compareTo is a static method in zencode but a virtual one in Java
+		boolean[] primitiveArguments = method.primitiveArguments;
```

Comment:
Huge workaround, we should find a better way to do this!

For ZC, `"".compareTo("")` is `compareTo("", "")` whereas the `method` believes it to be a virtual method.
So we get different-sized arrays here...